docker/README.md

```diff
@@ -0,0 +1,49 @@
+# FIWARE IoT Agent for the Ultralight 2.0 protocol
```

Comment:
A link to this README.md should be included in the main README.md (as it is done for instance in Orion repository, check https://github.com/telefonicaid/fiware-orion#advanced-topics)

Reply:
+1 to @fgalan, just a link should be enough

Reply:
This comment is yet pending, but given that PR #301 is not yet mature is this is probably about to merge, I'd suggest to move the comment to PR #301 so the link to docker/README.md be included there.

NTC